lib/fluent/plugin/out_amplitude.rb

```diff
@@ -119,6 +127,14 @@ def parse_time_from_string(time_string)
       log.info("failed to parse #{time_string}: #{e.message}")
     end
 
+    def extract_revenue_properties!(amplitude_hash, record)
+      REVENUE_PROPERTIES.each do |prop|
+        next unless record[prop]
```

Comment:
Should we not rather be using a `.nil?` check.  If we wanted to send the value `false`, it would exclude the key.